### ISSUE TITLE
[Feature-Request] Don't move clients out of swiss channels on disconnect

### ISSUE THREAD

**Issue by deluvas1911** [saltminede/saltychat-docs#33]
### Is your proposal related to a problem?
Not sure if it's a bug or an (currently) expected behaviour.

### Describe the feature you'd like
Currently the plugin is moving you out of swiss channels if you disconnect from the game server. It would be good if you only get moved out of the ingame channel.

### Sample case
1. Player joins the teamspeak, he's in the channel "**Welcome**"
2. Player joins the game server, he gets moved to the channel "**Ingame**" by the plugin
3. An supporter is moving the player from "**Ingame**" to "**Support**" (swiss channel) and asking him to leave the server because they have to talk
4. Player leaves the server and get's moved back to the channel "**Welcome**" instead of staying in the "**Support**" channel

#### Log of current behaviour
```
Salty Chat    |9  |Switched to a swiss channel
[... disconnecting from game server while being in a swiss channel ...]
Salty Chat    |9  |No pong command in the last 6 seconds...
Salty Chat    |9  |Switched channel from Support 1 to Welcome
```

**Comment by BlackFlash5** [comment]
Implemented in 2.3.2, feedback is appreciated.

**Comment by deluvas1911** [comment]
`Client is in a swiss channel or already in the original channel (AFK Team), no switch needed (Team 1)`
Seems good, thanks for the fast update
